rescript/package.json

```diff
@@ -1,23 +1,20 @@
 {
-  "name": "@catala-lang/french-law",
-  "version": "0.8.11",
+  "name": "@catala-lang/french-law-rescript",
```

Comment:
Does this mean we'll now only publish the `rescript` bindings for `french-law` and stop publishing a vanilla-JS-only npm package?

Reply:
Hm, no, there is actually:
- the base js (npm) packages that get generated by the `catala-examples` repo as `<pfx>/lib/catala-examples/french-law_npm.tar.gz` ; those could be published to npm, but should already become present on https://catalalang.github.io/catala/ (at the moment there is a tgz but that's not an installable npm package)
- the small rescript wrapper that is in `runtimes/rescript` and gets installed (simple copy) into `<pfx>/lib/catala/runtime_rescript` ; that directory can be `npm install`ed as well
- the package here that uses the two above to form a simpler rescript interface over the french-law computations

So the name change is to avoid conflict with the "pure js" version generated by `catala-examples`.
In theory this package definition should explicit dependencies towards `french-law`, but at the moment you should install that locally as pointed above, and `npm` would write the dependencies as pointers to the directories *on the local filesystem* which makes no sense in a git-published file.

Thankfully the Makefile should do the correct installations and work around this problem, but this will certainly need cleaning up and ensuring the website as published still works once things settle down.